public/coffee/ide/connection/ConnectionManager.coffee

```diff
@@ -94,7 +100,7 @@ define [], () ->
 			@ide.socket.on "connect_failed", () =>
 				@connected = false
 				$scope.$apply () =>
-					@$scope.state.error = "Unable to connect, please view the <u><a href='http://sharelatex.tenderapp.com/help/kb/latex-editor/editor-connection-problems'>connection problems guide</a></u> to fix the issue."
+					@$scope.state.error = "Unable to connect, please view the <u><a href='https://www.sharelatex.com/learn/Kb/Connection_problems'>connection problems guide</a></u> to fix the issue."
```

Comment:
Can we make this a relative url, i.e. `/learn/Kb/Connection_problems` so translation domains/local hosting is preserved.